src/index.ts

```diff
@@ -1,8 +1,8 @@
 export class C {
-    private x = 10
+    private x: number = 10
```

Comment:
This type annotation isn't really necessary even if it is to demonstrate functionality.